govatar.go

```diff
@@ -176,6 +176,7 @@ func mustAssetsList(dir string) []string {
 	assets := make([]string, len(dirEntries))
 	for i, dirEntry := range dirEntries {
 		assets[i] = filepath.Join(dir, dirEntry.Name())
+		assets[i] = filepath.ToSlash(assets[i])
```

Comment:
It's better to use `path.Join()` instead.

Reply:
Got it.